microsoft-edge/webdriver-chromium/index.md

```diff
@@ -60,6 +60,8 @@ To begin writing automated tests, make sure the Edge WebDriver version you insta
 
 1.  After the download completes, extract the `msedgedriver` executable to your preferred location. Add the folder where the executable is located to your `PATH` environment variable.
 
+    The driver executable needs to be placed in the PATH; see [IE Driver Server](https://www.selenium.dev/documentation/ie_driver_server/).  The top of that page reads: "The standalone server executable must be downloaded from the Downloads page and placed in your PATH."  If the driver location isn't included in the PATH, you must set the driver location at `webdriver.ie.driver` or some other way.
```

Comment:
Please make this update on the IE Mode page instead https://review.docs.microsoft.com/en-us/microsoft-edge/webdriver-chromium/ie-mode?branch=pr-en-us-1953&tabs=c-sharp

Reply:
@bwalderman - moved the paragraph(s) to other page